scripts/compute_flow_with_threads.py

```diff
@@ -0,0 +1,343 @@
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+import pickle
+import threading
+import time
+from datetime import datetime, timedelta
+
+from ocean_lib.assets.trusted_algorithms import add_publisher_trusted_algorithm
+from ocean_lib.config import Config
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.compute_input import ComputeInput
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.services.service import Service
+from ocean_lib.structures.abi_tuples import ConsumeFees, CreateErc20Data
+from ocean_lib.structures.file_objects import UrlFile
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def compute_flow(ocean: Ocean, wallet: Wallet, config: Config):
+    for _ in range(1000):
+        # Publish the data NFT token
+        DATA_nft_token = ocean.create_erc721_nft("NFTToken1", "NFT1", wallet)
+        assert DATA_nft_token.address
```

Comment:
Do you want to create the function in `script/compute_flow` and import in `scripts/compute_flow_with_threads` for avoiding duplication, right? If yes, then sure.

Reply:
@mariacarmina 

I see that the compute flow in `scripts` is copied from `tests/readmes/test_c2d_flow`. I think the function should be created where the logic originated. The function can then be called from `test_c2d_flow_readme` (in the same file) and from `tests/stress/compute_flow_with_threads.py`. 

I envision `compute_flow_with_threads.py` containing 1 test that uses `ThreadPoolExecutor` and is parameterized with the following arguments:

1. concurrent_flows - The number of thread pool workers
2. duration - The number of times each worker repeats the flow.

This would eliminate the need for the extra `scripts/compute_flow.py` file entirely.

This advice applies generally to the other scripts as well (consume, pool creation, and publish)